### ISSUE TITLE
stravelo icon for chrome bookmarks bar

### ISSUE THREAD

**Issue by rfschafer** [adam-conway/stavaelo#52]
Low priority to be sure, but seeing the stravelo icon on my chrome bookmarks bar would be cool. As an aside, it's a nice logo.

**Comment by adam-conway** [comment]
Yeah great idea! That’s not too hard to implement so I’ll try to add this soon. I believe we’ll have a different logo to use soon but making that also the favicon is a good point
